docs/pages/versions/unversioned/sdk/safe-area-context.md

```diff
@@ -51,3 +51,63 @@ class ClassComponent extends React.Component {
   }
 }
 ```
+
+## Migrating from CSS
+
+#### Before
+
+In a web-only app you would use CSS environment variables to get the size of the screen's safe area insets.
+
+`styles.css`
+
+```css
+div {
+  padding-top: env(safe-area-inset-top);
+  padding-left: env(safe-area-inset-left);
+  padding-bottom: env(safe-area-inset-bottom);
+  padding-right: env(safe-area-inset-right);
+}
+```
+
+#### After
+
+Universally, the hook `useSafeArea()` can provide access to this information.
+
+`App.js`
+
+```jsx
+import { useSafeArea } from 'react-native-safe-area-context';
+
+function App() {
+  const insets = useSafeArea();
+
+  return (
+    <View
+      style={{
+        paddingTop: insets.top,
+        paddingLeft: insets.left,
+        paddingBottom: insets.bottom,
+        paddingRight: insets.right,
+      }}
+    />
+  );
+}
+```
+
+### Web SSR
+
+If you are doing server side rendering on the web you can use `initialSafeAreaInsets` to inject insets value based on the device the user has, or simply pass zero values. Since insets measurement is async it will break rendering your page content otherwise.
```

Comment:
```suggestion
If you are doing server side rendering on the web, you can use `initialSafeAreaInsets` to inject values based on the device the user has, or simply pass zero. Otherwise, insets measurement will break rendering your page content since it is async.
```